plugin-infra/go-plugin-access/src/com/thoughtworks/go/plugin/access/scm/SCMConfiguration.java

```diff
@@ -0,0 +1,85 @@
+/*************************GO-LICENSE-START*********************************
+ * Copyright 2014 ThoughtWorks, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *************************GO-LICENSE-END***********************************/
+
+package com.thoughtworks.go.plugin.access.scm;
+
+
+import com.thoughtworks.go.plugin.api.config.Option;
+import com.thoughtworks.go.plugin.api.config.Options;
+import com.thoughtworks.go.plugin.api.config.Property;
+
+public class SCMConfiguration implements Comparable {
+
+    public static final Option<Boolean> REQUIRED = Property.REQUIRED;
+    public static final Option<Boolean> PART_OF_IDENTITY = Property.PART_OF_IDENTITY;
+    public static final Option<Boolean> SECURE = Property.SECURE;
+    public static final Option<String> DISPLAY_NAME = Property.DISPLAY_NAME;
+    public static final Option<Integer> DISPLAY_ORDER = Property.DISPLAY_ORDER;
```

Comment:
yes. `DISPLAY_NAME` & `DISPLAY_ORDER` will be used in material listing page to list properties & their values.